MainProject/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "MainProject": {
       "commandName": "Project",
-      "commandLineArgs": "--source=C:Images/image.jpg --output=kek --goal-format=.png"
+      "commandLineArgs": "--source=D://Studying//6_семестр//CG//MishaIsCringe//ImageConverter.Sdk//Images/bmp.bmp --output=D://Studying//6_семестр//CG//MishaIsCringe//ImageConverter.Sdk//Images/bmp2.bmp --goal-format=.bmp"
```

Comment:
For the future: let`s create a folder for test images

Reply:
You mean out of the project workspace? Smth like C:Images/image.jpg?

Reply:
I mean let's use a separate folder inside the project. And use relative path to it